sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/implementation/ModelsImpl.java

```diff
@@ -164,8 +165,11 @@ public Flux<ByteBuffer> getAsync(Boolean signed, Context context) {
     public InputStream get(Boolean signed) {
         Iterator<ByteBufferBackedInputStream> iterator =
                 getAsync(signed).map(ByteBufferBackedInputStream::new).toStream().iterator();
+        abstract class ModelReference implements Enumeration<InputStream> {
+        }
+
         Enumeration<InputStream> enumeration =
-                new Enumeration<InputStream>() {
+                new ModelReference() {
```

Comment:
We usually shouldn't be modifying Generated classes. Why was this required?

Reply:
This was required since in order to fix the spotcheck error: [ERROR] Low: The class com.azure.ai.personalizer.implementation.ModelsImpl$1 could be refactored into a named _static_ inner class [com.azure.ai.personalizer.implementation.ModelsImpl] At ModelsImpl.java:[line 167] SIC_INNER_SHOULD_BE_STATIC_ANON

Reply:
Will add a spot check exception instead as per recommendation.